pkg/services/pluginsintegration/pluginsintegration.go

```diff
@@ -144,6 +142,8 @@ var WireExtensionSet = wire.NewSet(
 	wire.Bind(new(plugins.Client), new(*backend.MiddlewareHandler)),
 	managedplugins.NewNoop,
 	wire.Bind(new(managedplugins.Manager), new(*managedplugins.Noop)),
+	sources.ProvideService,
```

Comment:
nothing new here, right?

Reply:
No, nothing new. It just tells wire to bind the interface to the concrete implementation in OSS, which is then overwritten in Enterprise: https://github.com/grafana/grafana-enterprise/blob/12e7cf221c17b62884ec5c9fce31a8adb924525e/src/pkg/extensions/pluginsintegration/pluginsintegration.go#L52-L54